### ISSUE TITLE
ET Open ruleset contains commented rules

### ISSUE THREAD

**Issue by regit** [StamusNetworks/scirius#50]
Emergingthreat files can contain rules that are commented. For example:
```
#Submitted by Joseph Gama
#Good rules, turn them on if you are interested. They are accurate.
#
#alert tcp $EXTERNAL_NET any -> $HOME_NET any (msg:"ET POLICY PE EXE or DLL Windows file download"; flow:established,to_client; content:"MZ"; byte_jump:4,58,relative,little; content:"PE|00 00|"; distance:-64; within:4; flowbits:set,ET.http.binary; reference:url,doc.emergingthreats.net/bin/view/Main/2000419; classtype:policy-violation; sid:2000419; rev:22;)
```
It could be interesting to be able to parse them and maybe to add them to disabled rules.

Problem found by @b-u-g-s in https://github.com/StamusNetworks/SELKS/issues/26

**Comment by pevma** [comment]
I think it is a good idea to be able to enable all rules present in a rulefile. The end user however needs to be aware there could potentially be unwanted consequences  -  aka duplicates, perf hit, FPs etc.

**Comment by regit** [comment]
I don't think enabling commented rules is a good idea. There will be too much noise. My idea is to:
* parse them
* add them to categories
* add them to disabled rules
This way we get exactly what rules writer wanted to do.

**Comment by b-u-g-s** [comment]
Great idea! This would work great and be very true to what actually really happens... Those rules are disabled!

**Comment by b-u-g-s** [comment]
A few thoughts:
- maybe this is the case already but do you display disable rules by categories you can drill down? I think there are quite a few rules that are commented so you wouldn't just want to list individual rules as a flat list. 
- I noticed some disabled rules have double # in front. So parsing only the rules with one # would not be enough

On the last point, this is why I didn't notice the rules were disabled with the original issue (windows download). There were so many lines starting with one # that I thought it was something like #include in C and that you had to have double # for disabling a rule (nothing to do with C)

**Comment by regit** [comment]
Implemented in 1.1.2.
